node/src/components/linear_chain_sync.rs

```diff
@@ -0,0 +1,981 @@
+//! Linear chain synchronizer.
```

Comment:
This file has been removed on `feat-fast-sync`, I believe we don't want to re-introduce it.

Reply:
Yes, thanks for catching these.